### PR TITLE
Add demos to ordered layout demos

### PR DESCRIPTION
**Pull request by shadikhani**
Fixes https://github.com/vaadin/90min-dx-tasks/issues/134#issuecomment-629115683

### CODE REVIEW

**Comment by tmattsso** [comment]
With these new changes, now the FlexLayout tab is not aligned visually; it uses buttons instead of radiogroup for the settings.

**Comment by tmattsso** [comment]
Box-size demos are not clear what the setting actually does.

**Comment by shadikhani** [comment]
> Box-size demos are not clear what the setting actually does.



> With these new changes, now the FlexLayout tab is not aligned visually; it uses buttons instead of radiogroup for the settings.

I think we can have these demo examples here, and your examples in the layout examples.

File: vaadin-ordered-layout-flow-demo/src/main/java/com/vaadin/flow/component/orderedlayout/demo/OrderedLayoutView.java [hunk 1]
```diff
@@ -699,6 +914,160 @@ private void createVerticalLayoutWithExpandingContent() {
...
 
+    private void verticalLayoutWithScrolling() {
+        // begin-source-example
+        // source-example-heading: Vertically scrolling
+        VerticalLayout layout = new VerticalLayout();
+        layout.setPadding(true);
+        layout.setHeight("300px");
+
+        Component component1 = createComponent(1, "#78909C");
+        // necessary so that the layout doesn't try to shrink the button.
```
Comment (tomivirkki) [inline vaadin-ordered-layout-flow-demo/src/main/java/com/vaadin/flow/component/orderedlayout/demo/OrderedLayoutView.java:925]: This is one of the reasons why the Scroller was built (examples in the same file). Let's not add a demo with hacks for a use-case we already have a built-in component for.

Comment (shadikhani) [reply vaadin-ordered-layout-flow-demo/src/main/java/com/vaadin/flow/component/orderedlayout/demo/OrderedLayoutView.java]: Done

**Comment by anasmi** [comment]
@tomivirkki Could you review it as you have checked the code before ? :)

**Comment by tomivirkki** [comment]
@anasmi 👍  I can take a look once you've finished the internal review (no approvals from anyone yet); you probably have the best understanding of the demo redesign.

**Pull request merged**